### PR TITLE
feat(pattern/flags): add usage doc

### PR DESCRIPTION
**Pull request by caqueste**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: MZC-104

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

**Comment by caqueste** [comment]
🚀 **Deployement url :** https://patternflagsdesign-dot-demo-dot-mozaic-design-system-dtrp.appspot.com

**Comment by tiagoux** [comment]
~partenaire~

File: src/pages/Components/Flags/index.md [hunk 1]
```diff
@@ -3,40 +3,54 @@ title: 'Flags'
 order: 10
 ---
 
-> Definition
+> A flag is used to visually categorize an **invariable** content for a given period.
```
Comment (gael-boyenval) [inline src/pages/Components/Flags/index.md]: what ? invariable ?

File: src/pages/Components/Flags/index.md [hunk 2]
```diff
@@ -3,40 +3,54 @@ title: 'Flags'
...
-### Variation 1
+## Variations
 
-<preview path="src/pages/src/pages/Components/Flags/previews/Flags" nude="true"></preview>
+-- Preview with the all flags --
 
-### Variation 2
+| Color Scheme   | Usage                                                 |
+| -------------- | ----------------------------------------------------- |
+| Primary        | Used to categorize a content with Leroy Merlin        |
```
Comment (gael-boyenval) [inline src/pages/Components/Flags/index.md]: all those usages are unclear to me

**Pull request merged**